src/WebJobs.Extensions.DurableTask/ProtobufUtils.cs

```diff
@@ -97,7 +97,11 @@ public static P.HistoryEvent ToHistoryEventProto(HistoryEvent e)
                             },
                         },
                         ScheduledStartTimestamp = startedEvent.ScheduledStartTime == null ? null : Timestamp.FromDateTime(startedEvent.ScheduledStartTime.Value),
-                        CorrelationData = startedEvent.Correlation,
+                        ParentTraceContext = startedEvent.ParentTraceContext == null ? null : new P.TraceContext
+                        {
+                            TraceParent = startedEvent.ParentTraceContext.TraceParent,
+                            TraceState = startedEvent.ParentTraceContext.TraceState,
+                        },
```

Comment:
Just confirming - will distributed tracing V1 customers be affected by this? As in - will their distributed tracing telemetry possibly break?

Reply:
This code path is currently used by .NET Isolated and Java. I took a look at the codebases for both of these SDKs, and they don't appear to be using `CorrelationData` (searched the whole repo for it) so I expect this should be non-breaking.